e2e/app/page/notebook-cell.ts

```diff
@@ -0,0 +1,152 @@
+import {ElementHandle, Frame, Page} from 'puppeteer';
+import {getPropValue} from 'utils/element-utils';
+
+export enum CellType {
+  // To append to css selector
+  Code = '.code_cell',
+  Markdown = '.text_cell',
+  Any = '',
+}
+
+export default class NotebookCell {
+
+  private readonly page: Page;
+  private iframe: Frame; // Jupyter notebook in iframe
+  private cellIndex: number;
```

Comment:
The constructor indicates this is option, but the type here does not. Which should it be?